src/Core/Worker.php

```diff
@@ -303,7 +331,11 @@ private static function execFunction($queue, $funcname, $argv) {
 		// Reset global data to avoid interferences
 		unset($_SESSION);
 
-		$funcname($argv, $argc);
+		if ($method_call) {
```

Comment:
Can't `$method_call` be the `method_exists(sprintf('Friendica\Worker\%s', $include), 'execute')` condition here, removing the need for an extra parameter?

Reply:
This is just temporarily, until the last old files had been removed. I don't see much need to make it perfect for this short time.

Reply:
I'm weary of "this is just temporary" because we aren't working full-time on it, so there's no clear horizon for temporary code deletion. I'd suggest to do it well regardless because we never know when it's going to be finished anyway.